### PR TITLE
feat: add scroll to the selected option

### PR DESCRIPTION
**Pull request by irekrog**
* add scrollToSelectedOption prop
* add an example

### CODE REVIEW

**Review by koko57 — APPROVED** [review]

**Review by gitsad — APPROVED** [review]

File: src/types/index.ts [hunk 1]
```diff
@@ -165,6 +165,13 @@ export interface SelectProps {
      * @default "Select..."
      */
     placeholderText?: string;
+    /**
```
Comment (gitsad) [inline src/types/index.ts:168]: Don't you need generate typedoc again?

**Comment by gitsad** [comment]
I wanted simply request changes, because I have wrongly approved it 🤷  Don't you need regenerate typedoc here?

**Comment by gitsad** [comment]
> I wanted simply request changes, because I have wrongly approved it 🤷 Don't you need regenerate typedoc here?

Ok I see that [here](https://github.com/MobileReality/react-native-select-pro/pull/26) it will be automated

File: src/types/index.ts (hunk 1 above)
Comment (irekrog) [reply src/types/index.ts:168]: No, I don't want because I want to generate new docs after release new version to npm

**Comment by irekrog** [comment]
> > I wanted simply request changes, because I have wrongly approved it 🤷 Don't you need regenerate typedoc here?
> 
> Ok I see that [here](https://github.com/MobileReality/react-native-select-pro/pull/26) it will be automated

Yep

**Pull request merged**